vzstorage-pd/virtuozzo-provisioner.go

```diff
@@ -277,6 +278,16 @@ func (p *vzFSProvisioner) Delete(volume *v1.PersistentVolume) error {
 	}
 
 	path := mount + "/" + options["volumePath"] + "/" + options["volumeID"]
+
+	mounts, err := vstorage.ReadMounts("/proc/mounts")
```

Comment:
ploop-volume solves this issue, in alpha provisioner version I decided to just read image file from disk descriptor xml and try to remove it first. That should do the trick